docs/conf.py

```diff
@@ -91,6 +91,7 @@
     "common/craft-parts/explanation/how_parts_are_built.rst",
     "common/craft-parts/explanation/parts.rst",
     "common/craft-parts/explanation/dump_plugin.rst",
+    "common/craft-parts/explanation/gradle_plugin.rst",
```

Comment:
@medubelko - what do you want to do with this?  I see you ignored the dump plugin explanation.